### PR TITLE
Fixed LiveScript scaffold test and added support for scaffolding app as ES6

### PR DESCRIPTION
**Pull request by gregstewart**
Added an option to scaffold the Arch app as ES6. It still defaults to LiveScript, but I believe that if we want to increase adoption, involvement and usage of the framework we should offer a JS/ES6 solution as well.

### CODE REVIEW

File: app/parts/02_prompts.ls [hunk 1]
```diff
@@ -14,6 +14,11 @@ module.exports =
         message: 'Project directory'
         default: @options.dir or '.'
 
+      unless @options.scaffold_as => prompts.push do
+        name: 'scaffold'
```
Comment (charypar) [inline app/parts/02_prompts.ls]: should we call that option `language`? If I understand correctly what this does :)

**Comment by charypar** [comment]
This is awesome!

File: app/parts/02_prompts.ls (hunk 1 above)
Comment (gregstewart) [inline app/parts/02_prompts.ls]: Happy to change it to `language`.

**Comment by StuartHarris** [comment]
@gregstewart nice one! I started doing the same thing at the weekend, but was full of cold and feeling shit so didn't get very far. Shall we open an issue to debate the pros and cons of porting the code to ES6 too? :boom: Someone will sooner or later :smile:

**Comment by charypar** [comment]
@StuartHarris :fire:

**Comment by gregstewart** [comment]
Well IMO the framework can be written in whatever language you want. Using the framework should not prescribe what language to use. There's a case to be made for moving to more `vanilla` JS if you want more community involvement. I guess there's no harm in having a discussion about it.

**Comment by charypar** [comment]
Of course there is no harm :) it's more of a running joke me defending LiveScript and Stu pushing for ES6 :)

And I absolutely agree that the framework shouldn't dictate a language. I think it can strongly suggest one, but as with anything, provide an out for people who just don't want to or can't.

**Comment by tabazevedo** [comment]
Can this be merged @charypar @gregstewart ?

**Pull request merged**